README.md

```diff
@@ -5,8 +5,15 @@
 [![Join the chat on https://gitter.im/exonum/exonum-java-binding](https://img.shields.io/gitter/room/exonum/exonum-java-binding.svg?label=Chat)](https://gitter.im/exonum/exonum-java-binding)
 
 This is a primary repository for Exonum projects in Java.  
+[Exonum][exonum] is an extensible open-source framework for creating blockchain applications.
+
+## Projects
 The following projects are available:
- - [Exonum Java Binding](exonum-java-binding-parent)
+ - [Exonum Java Binding](exonum-java-binding-parent). It is a framework for building blockchain 
+ applications in Java which provides Exonum services runtime environment
+ - [Light Client](exonum-java-light-client). It is a library for interaction with Exonum 
```

Comment:
"It is a library for interaction with Exonum (blockchain?) from client (or external?) applications."

I'd consider putting a more detailed _and_ complete description in the corresponding README, not here (digital signatures seem irrelevant).